doc/getting_started/up2.rst

```diff
@@ -1,9 +1,10 @@
-==========================================
-ACRN - Getting started guide for UP2 board
-==========================================
+.. _getting_started_up2:
+
+Getting started guide for UP2 board
+###################################
 
 Hardware setup
-==============
+**************
 
 The `*UP squared board* <http://www.up-board.org/upsquared/>`__ (UP2) is
```

Comment:
I notice all the links on this page are being emphasized inconsistent with the rest of the documentation.  Change them all to not use an asterisk, e.g.,

    `UP squared board <http://www.up-board.org/upsquared/>`_